src/prefect/tasks/github/comments.py

```diff
@@ -0,0 +1,86 @@
+import json
+from typing import Any
+
+from prefect import Task
+from prefect.utilities.tasks import defaults_from_attrs
+
+
+class CreateIssueComment(Task):
+    """
+    Task for creating a comment on the specified GitHub issue using the v3 version of the
+    GitHub REST API.
+
+    Args:
+        - repo (str, optional): the name of the repository to create the comment in; must be
+            provided in the form `organization/repo_name`; can also be provided to the `run`
+            method
+        - issue_number (int, optional): the ID of the issue to create the comment on.
+        - body (str, optional): the body of the comment; can also be provided to the `run` method
+        - **kwargs (Any, optional): additional keyword arguments to pass to the standard Task
+            init method
+
+    References:
+        - https://developer.github.com/v3/issues/comments/#create-an-issue-comment
+    """
+
+    def __init__(
+        self,
+        repo: str = None,
+        issue_number: int = None,
+        body: str = None,
+        **kwargs: Any
+    ):
```

Comment:
Should I use `Optional` here?

```suggestion
    def __init__(
        self,
        repo: Optional[str] = None,
        issue_number: Optional[int] = None,
        body: Optiona[str] = None,
        **kwargs: Any
    ):
```

Reply:
I believe mypy does this automatically, I think what you have here is fine.